spec/feature_tests_spec.rb

```diff
@@ -0,0 +1,55 @@
+# As a customer
+# So that I can check if I want to order something
+# I would like to see a list of dishes with prices
+require 'takeaway'
+require 'order'
+
+describe "TakeAway features" do
+  it "can see a list of dishes with prices" do
+    takeaway = TakeAway.new
+    expect(takeaway.view_dishes).to eq("Hamburger £4, Hotdog £3, Pizza £5, Fried chicken £3")
+  end
+end
+
+# As a customer
+# So that I can order the meal I want
+# I would like to be able to select some number of several available dishes
+
+describe "Takeaway features" do
+  it "a customer can add an order of several available dishes" do
+    takeaway = TakeAway.new
```

Comment:
can you not use subject?

also if you put 'subject(:takeaway) { described_class.new }' then this renames subject for you throughout the entire spec file